specs/error_state/ErrorInvalidOpcode.md

```diff
@@ -0,0 +1,21 @@
+# ErrorInvalidOpcode state
+
+## Procedure
+
+`ErrorInvalidOpcode` may occur when invoked via `CALL`, `CALLCODE`, `DELEGATECALL`, `STATICCALL`, `CREATE` and `CREATE2`.
```

Comment:
I think invalid opcode could happen in any step, why specially these opcodes here?

Reply:
Fixed in [08d7b18](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/362/commits/08d7b1887a186e5ad494bb89625598e2f7c4bcaa) as:
```
`ErrorInvalidOpcode` may occur when an invalid opcode is encountered in any step during the execution.
```

I found this error is returned by [opUndefined function](https://github.com/ethereum/go-ethereum/blob/master/core/vm/instructions.go#L811) and it is set to [JumpTable code](https://github.com/ethereum/go-ethereum/blob/master/core/vm/jump_table.go#L1048) in go-ethereum.
It could return in [evm.interpreter.Run code](https://github.com/ethereum/go-ethereum/blob/master/core/vm/interpreter.go#L236) which could be invoked by `CALL`, `CALLCODE`, `DELEGATECALL`, `STATICCALL`, `CREATE` and `CREATE2`.

I am sorry that I misunderstood the [for loop](https://github.com/ethereum/go-ethereum/blob/master/core/vm/interpreter.go#L178) in `evm.interpreter.Run`. This error should be returned when encountering an invalid opcode.

Reply:
Got it, thanks for clarifying.